docs/api-reference/time-series-scorer.md

```diff
@@ -0,0 +1,26 @@
+### Anomaly Scorer
+Once the raw score at a timestamp is computed, it is fed to the anomaly scorer component to calculate the final anomaly score at that timestamp.
+There are two statistics involved in this scorer, p-value and martingale score.
+
+#### Spike detection based on p-value
+The p-value score indicates the p-value of the current computed raw score according to a distribution of raw scores.
+Here, the distribution is estimated based on the most recent raw score values up to certain depth back in the history.
+More specifically, this distribution is estimated using [kernel density estimation](https://en.wikipedia.org/wiki/Kernel_density_estimation)
+with the Gaussian [kernels](https://en.wikipedia.org/wiki/Kernel_(statistics)#In_non-parametric_statistics) of adaptive bandwidth.
```

Comment:
>(statistics) [](start = 65, length = 12)

is the link correct? does this need to go?

Reply:
It is correct. Everything not learnt by .NET developer needs a reference.